LabExT/Experiments/StandardExperiment.py

```diff
@@ -114,6 +114,29 @@ def read_parameters_to_variables(self):
         self.param_output_path = str(self.save_parameters["Raw output path"].value)
         makedirs(self.param_output_path, exist_ok=True)
 
+    def ask_user_to_continue_even_if_live_viewer_active(self):
+        """ check if any of the instrument addresses in the ToDo queue are currently active in live viewer """
```

Comment:
Here in the docstring we could add the return values of the function, but I don't think it's very important.